docs/source/user_guide/causality_intro.rst

```diff
@@ -1,10 +0,0 @@
-Introduction to causality
-=========================
-DoWhy is based on a simple unifying language for causal inference, unifying two
-powerful frameworks: 
-causal graphs  and potential outcomes. It uses graph-based criteria and
-do-calculus for modeling assumptions and identifying a non-parametric causal effect. 
-For estimation, it switches to methods based primarily on potential outcomes.
-
-
-For a quick introduction to causal inference, check out `amit-sharma/causal-inference-tutorial <https://github.com/amit-sharma/causal-inference-tutorial/>`_. We also gave a more comprehensive tutorial at the ACM Knowledge Discovery and Data Mining (`KDD 2018 <http://www.kdd.org/kdd2018/>`_) conference: `causalinference.gitlab.io/kdd-tutorial <http://causalinference.gitlab.io/kdd-tutorial/>`_. For an introduction to the four steps of causal inference and its implications for machine learning, you can access this video tutorial from Microsoft Research: `DoWhy Webinar <https://note.microsoft.com/MSR-Webinar-DoWhy-Library-Registration-On-Demand.html>`_.
```

Comment:
For the user-guide, we may have a page that refers to useful links to learn more about causal inference.
This rst file was a stub intended to be expanded in that direction. 

In the new docs, where do you think that belongs? Shall we link to it in the first page of user_guide, saying, "If you'd to learn more about causal inference concepts, here are some resources [link to page]" ?

also it will be nice to mention somewhere about potential outcomes and graphs, as the text in this file does.

Reply:
>In the new docs, where do you think that belongs? Shall we link to it in the first page of user_guide, saying, "If you'd to learn more about causal inference concepts, here are some resources [link to page]" ?

Yes, that's a good idea. I wasn't clear about your intentions here and so this helps. Let change this accordingly.

Reply:
This part now found its way into the Getting Started section in a slightly different format. Please have a look.